src/blocks/Promo/Promo.js

```diff
@@ -30,32 +32,27 @@ const PromoMain = styled('div', {
 }))
 
 function Promo(props) {
-  const { name, description, subtitle, imageSrc, alignContent, imageCircle } = props
+  const { name, description, subtitle, mediaProps, alignContent = 'left', imageCircle } = props
+
+  const circleSettings = '60px at 50%'
 
   return (
     <PromoRoot>
       <PromoMain>
-        <Card sx={{ display: 'flex', background: '#E9E9E9' }}>
-          {alignContent === 'right' && (
-            <CardMedia
-              component="img"
-              image={imageSrc}
-              alt={name}
-              sx={{
-                width: '300px',
-                clipPath: () => {
-                  if (imageCircle) {
-                    return 'circle(100px at 50%)'
-                  }
-                  return 0
-                },
-              }}
-            />
-          )}
-
-          <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+        <Card
+          sx={{
+            display: 'flex',
+            bgcolor: 'secondary.light',
+            flexDirection: alignContent === 'right' ? 'row-reverse' : 'row',
+            '@media(max-width: 900px)': {
```

Comment:
please use the theme based breakpoints:   `
[theme.breakpoints.up('SCREENSIZE')]: {
    .................
  },
`

instead of max-width